examples/tree_2d_dgsem/elixir_euler_positivity.jl

```diff
@@ -101,13 +101,12 @@ callbacks = CallbackSet(summary_callback,
 limiter! = PositivityPreservingLimiterZhangShu(thresholds=(5.0e-6, 5.0e-6),
                                                variables=(Trixi.density, pressure))
 stage_limiter! = limiter!
-step_limiter!  = limiter!
 
 
 ###############################################################################
 # run the simulation
 
-sol = solve(ode, CarpenterKennedy2N54(stage_limiter!, step_limiter!, williamson_condition=false),
```

Comment:
This is unrelated to deprecations right? Is this just not necessary because the step limiter makes it redundant?